CONTRIBUTING.md

```diff
@@ -43,8 +42,8 @@ directly (excluding inital commits on new interface/util repositories) to master
 comment) or merged by the PSR Editor, except in the review phase when the coordinator
 should seek commment from the editor, but merging is at the coordinator's discretion
 * You must never merge a pull request that affects any file in the repository
-other than those you are on a working group for; you should request a secretary
-or member of that working group (mention @php-fig/psr-x) do so
+other than those you are on a working group for unless you are a secretary; you should
+request a secretary or member of that working group (mention @php-fig/psr-x) do so
```

Comment:
before, it was implicitly obvious, now its more complicated. or is that a change of policy that secretaries are now allowed to merge things they have not been allowed to? maybe split it into "Secretaries are allowed to merge any pull requests" and "Editors and sponsors may only merge pull requests that exclusively affects files of their working group(s); for other pull requests, they should request..."

Reply:
IMHO that specification is not needed; it's basically implicit because the secretaries have by definition admin rights on any PHP-FIG repo, so they act as gatekeepers. There are bylaws in place that already define what or what not a secretary may merge without second guessing... Basically just typos and the like, see: https://www.php-fig.org/bylaws/psr-amendments/#32-formatting--typos